thumbor/optimizers/gifv.py

```diff
@@ -15,20 +15,28 @@
 
 from thumbor.optimizers import BaseOptimizer
 from thumbor.utils import logger
+from os.path import exists
 
 
 class Optimizer(BaseOptimizer):
 
     def should_run(self, image_extension, buffer):
-        return 'gif' in image_extension and 'gifv' in self.context.request.filters
+        if 'gif' in image_extension and 'gifv' in self.context.request.filters:
+            if not exists(self.context.config.FFMPEG_PATH):
+                logger.warn(
+                    'gifv optimize enabled but binary JPEGTRAN_PATH does not exist')
```

Comment:
Did you mean FFMPEG_PATH?

Reply:
Also gifv optimizeR, instead of optimize.

Reply:
Fixed!!! Sorry, my mistake!!!